frontend/components/dashboard/SideBar.tsx

```diff
@@ -28,6 +29,10 @@ interface OverrideProps {
   pos: number;
   comment: string;
 }
+export interface DeleteRowFunctionProps { 
+  ids: string[]; 
```

Comment:
I centralize this type. I noticed an inconsistency in the data property. In all cases, this is one item, why is it an array and not an object?

Reply:
This is very strange. Logically, it should be an ObjectId, I don't know why it is an array

Reply:
I made some refactoring in the sidebar component. maybe I can create a pr with this change for your review?

Reply:
@JoaoVictor6 Sounds good! Looking forward